### PR TITLE
ZipSlip C# demo

### PR DESCRIPTION
**Pull request by calumgrant**

### CODE REVIEW

File: ql_demos/csharp/ZipSlip/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Use [this snapshot](http://downloads.lgtm.com/snapshots/csharp/microsoft/powershell/PowerShell_PowerShell_csharp-srcVersion_450d884668ca477c6581ce597958f021fac30bff-dist_odasa-lgtm-2018-09-11-e5cbe16-linux64.zip)
+of PowerShell.
+
+## Introduction
+
+The "Zip Slip" vulnerability was announced on June 5th 2018, by [Snyk](https://snyk.io/research/zip-slip-vulnerability).
+You can see on [this page](https://snyk.io/research/zip-slip-vulnerability#dot-net) some sample code that shows the vulnerable code.
+
+Microsoft immediately wanted to search their codebase to see if any of their own code was vulnerable. Within a few days,
+they had written a basic query and run it on their own LGTM-e instances that turned up several results across
```
Comment (tibbes) [inline ql_demos/csharp/ZipSlip/README.md]:
```suggestion
they had written a basic query and run it on their own LGTM Enterprise instances that turned up several results across
```

File: ql_demos/csharp/ZipSlip/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+several codeases. Because Semmle has a close working relationship with Microsoft, we then helped Microsoft to refine
+that query further and submit it as a [pull request](https://github.com/Semmle/ql/pull/54) against our open source QL repository.
+
+It was deployed within 2 weeks where it was run over thousands of open source C# projects.
```
Comment (tibbes) [inline ql_demos/csharp/ZipSlip/README.md]:
```suggestion
It was deployed to [LGTM.com](https://lgtm.com) within 2 weeks, where it was run over thousands of open source C# projects.
```

File: ql_demos/csharp/ZipSlip/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Here are some [sample results](https://lgtm.com/rules/1506511188430/alerts/) for the ZipSlip query. 
+One of those projects was Microsoft PowerShell.
+
+As a result of this query, [a senior Microsoft engineer](https://github.com/TravisEz13)
+fixed this query in November 2018 in
```
Comment (kevinbackhouse) [inline ql_demos/csharp/ZipSlip/README.md]: query ==> vulnerability

**Comment by calumgrant** [comment]
All comments addressed and ready to merge.

**Pull request merged**